recipe/install_nvcc.sh

```diff
@@ -127,6 +127,6 @@ EOF
 mkdir -p "${PREFIX}/bin"
 cat > "${PREFIX}/bin/nvcc" <<EOF
 #!/bin/bash
-"\${CUDA_HOME}/bin/nvcc" -ccbin "\${CXX}" \$@
+"\${CUDA_HOME}/bin/nvcc" -ccbin "\${CXX}" -I\${CONDA_PREFIX}/include \$@
```

Comment:
This points to the build prefix in conda-build.

Reply:
Notice the backslash: `\${CONDA_PREFIX}`

Reply:
and ?

Reply:
Are you intending to add `$BUILD_PREFIX/include` when this package is used as a compiler in conda-build?

Reply:
Yes, the application is about using nvcc inside a conda-build of another recipe.
IIUC, the `include` prefix should be the same as the prefix of `$CXX`.

Reply:
That's wrong. `BUILD_PREFIX` (`CONDA_PREFIX`) is for build tools in the build system. like the compiler, cmake, make etc. `PREFIX` is for the host tools where the libraries are located.

Reply:
So, when nvcc is run in conda-build, the prefix should be `\${PREFIX}`.
And when nvcc is run in user's conda environment, the prefix should be `\${CONDA_PREFIX}`.
Is this correct?
If not, my next guess would be to use `${PREFIX}` (the conda prefix transformations are still confusing to me).

Reply:
Considering `nvcc` is a compiler and we likely will install it into the `BUILD_PREFIX` should we include the `BUILD_PREFIX` first (if it exists) and then include `PREFIX`?

Reply:
I think what @pearu [said above]( https://github.com/conda-forge/nvcc-feedstock/pull/33#discussion_r347106624 ) was correct. Namely `PREFIX` in the build and `CONDA_PREFIX` outside of a build.

Should add we can check whether `CONDA_BUILD` is set to `1` to determine if we are in a conda-build context.

ref: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process

Reply:
@isuruf, is there a way we could bake the include path into [ctng-compiler-activation]( https://github.com/conda-forge/ctng-compiler-activation-feedstock )?